ecosystem/indexer-grpc/indexer-grpc-file-store/src/blob_data.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub struct TransactionsBlob {
+    /// The version of the first transaction in the blob.
+    pub starting_version: u64,
+    /// The transactions in the blob.
+    pub transactions: Vec<String>,
+}
```

Comment:
This can go into processor.rs directly I think. No reason to have a separate file for just a struct.

Reply:
Done.